src/blocks/modal.css

```diff
@@ -177,17 +283,8 @@
 }
 
 .modal__button-submit_disabled {
-  font-family: "Poppins", Arial, Helvetica, sans-serif;
-  font-weight: 500;
-  font-size: 18px;
-  line-height: 1;
-  color: #fcf5e5;
-  background-color: #212121;
   opacity: 50%;
-  border-radius: 8px;
-  margin: 0 0 32px 50px;
-  width: 156px;
-  height: 52px;
+  cursor: not-allowed;
 }
 
 @media screen and (max-width: 627px) {
```

Comment:
Please, add ` max-width: 276px;` for `.modal__input` in line 312 of `modal.css`, to make the inputs match the `mobile` design https://snipboard.io/a8Lb94.jpg

```css

  .modal__input {
    font-size: 16px;
    margin: 0;
    max-width: 276px;
  }
```